apps/calendar/js/controllers/notifications.js

```diff
@@ -55,26 +55,27 @@ function issueNotification(alarm) {
     }
 
     var begins = calc.dateFromTransport(busytime.start);
-    var distance = dateFormat.fromNow(begins);
-    var now = new Date();
+    return dateFormat.relativeDate(begins).then(distance => {
```

Comment:
ok. Is there a way for me to wrap the Promise returned by `formatter.format()` in it, or do I need to write some shim?
Sorry, I'm a bit lost on how to transition between promises and async/await.